cmd/kops/create.go

```diff
@@ -147,6 +150,23 @@ func RunCreate(f *util.Factory, out io.Writer, c *CreateOptions) error {
 		}
 
 	}
-
+	{
+		var sb bytes.Buffer
+		fmt.Fprintf(&sb, "\n")
+		fmt.Fprintf(&sb, "Your input has been successfully parsed.\n")
```

Comment:
I think parsed is not quite the right verb.  We have created the objects in kops, we just haven't applied the changes to AWS.

One option could be to keep a list of the objects created (or just print as we go) and say

```
Created cluster/mycluster object
Created instancegroup/nodes object

To deploy these resources, run: kops update cluster %s --yes
```

I do like the "deploy" verb - probably more descriptive than "update", in fact...

Otherwise, looks good!